composer.json

```diff
@@ -66,7 +66,7 @@
     },
     "require-dev": {
         "helmich/phpunit-json-assert": "^3.0",
-        "phpunit/phpunit": "^8.2.5",
+        "phpunit/phpunit": "^8.2.5 <8.4",
```

Comment:
For future reference, this is due to https://github.com/symfony/symfony/issues/34231